stable/unifi/templates/controller-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "unifi.fullname" . }}-controller
+  labels:
+    app: {{ template "unifi.name" . }}
+    chart: {{ template "unifi.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.controllerService.type }}
+  ports:
+    - port: {{ .Values.controllerService.port }}
+      targetPort: controller
+      protocol: TCP
+      name: controller
```

Comment:
If a specific nodePort is defined in values.yml for the controller, nothing would pick that up as there is no check for it here.

Reply:
fixed!